### PR TITLE
Merge sorted list

### PR DESCRIPTION
**Pull request by joankovacs**
uhm uhhhh uh please review this code uhhhhh

### CODE REVIEW

File: functions/merge_sorted_lists.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 def merge_sorted_lists(l1, l2):
-    pass
```
Comment (AmberRose2) [inline functions/merge_sorted_lists.py:2]: Nice job passing up on pass!

File: tests/merge_sorted_lists_test.py [hunk 2]
```diff
@@ -1,2 +1,37 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
+
```
Comment (AmberRose2) [inline tests/merge_sorted_lists_test.py:3]: This looks sufficiently sexy

**Review by AmberRose2 — APPROVED** [review]
PLZ tattoo me.

File: tests/merge_sorted_lists_test.py (hunk 2 above)
Comment (joankovacs) [reply tests/merge_sorted_lists_test.py:3]: thanks i'm an expert

**Pull request merged**